test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -5696,29 +5696,32 @@ BOOST_AUTO_TEST_CASE(strip_reason_strings)
 			}
 		}
 	)";
-	m_revertStrings = RevertStrings::Default;
-	compileAndRun(sourceCode, 0, "C");
+	ALSO_VIA_YUL(
+		DISABLE_EWASM_TESTRUN()
+		m_revertStrings = RevertStrings::Default;
+		compileAndRun(sourceCode, 0, "C");
 
-	if (
-		m_optimiserSettings == OptimiserSettings::minimal() ||
-		m_optimiserSettings == OptimiserSettings::none()
-	)
-		// check that the reason string IS part of the binary.
-		BOOST_CHECK(toHex(m_output).find("736f6d6520726561736f6e") != std::string::npos);
+		if (
+			m_optimiserSettings == OptimiserSettings::minimal() ||
+			m_optimiserSettings == OptimiserSettings::none()
+		)
+			// check that the reason string IS part of the binary.
+			BOOST_CHECK(toHex(m_output).find("736f6d6520726561736f6e") != std::string::npos);
 
-	m_revertStrings = RevertStrings::Strip;
-	compileAndRun(sourceCode, 0, "C");
-	// check that the reason string is NOT part of the binary.
-	BOOST_CHECK(toHex(m_output).find("736f6d6520726561736f6e") == std::string::npos);
-
-	ABI_CHECK(callContractFunction("f(bool)", true), encodeArgs(7));
-	ABI_CHECK(callContractFunction("f(bool)", false), encodeArgs());
-	ABI_CHECK(callContractFunction("g(bool)", true), encodeArgs(8));
-	ABI_CHECK(callContractFunction("g(bool)", false), encodeArgs());
-	ABI_CHECK(callContractFunction("f1(bool)", true), encodeArgs(9));
-	ABI_CHECK(callContractFunction("f1(bool)", false), encodeArgs());
-	ABI_CHECK(callContractFunction("g1(bool)", true), encodeArgs(10));
-	ABI_CHECK(callContractFunction("g1(bool)", false), encodeArgs());
+		m_revertStrings = RevertStrings::Strip;
+		compileAndRun(sourceCode, 0, "C");
+		// check that the reason string is NOT part of the binary.
+		BOOST_CHECK(toHex(m_output).find("736f6d6520726561736f6e") == std::string::npos);
+
+		ABI_CHECK(callContractFunction("f(bool)", true), encodeArgs(7));
+		ABI_CHECK(callContractFunction("f(bool)", false), encodeArgs());
+		ABI_CHECK(callContractFunction("g(bool)", true), encodeArgs(8));
+		ABI_CHECK(callContractFunction("g(bool)", false), encodeArgs());
+		ABI_CHECK(callContractFunction("f1(bool)", true), encodeArgs(9));
+		ABI_CHECK(callContractFunction("f1(bool)", false), encodeArgs());
+		ABI_CHECK(callContractFunction("g1(bool)", true), encodeArgs(10));
+		ABI_CHECK(callContractFunction("g1(bool)", false), encodeArgs());
+	)
```

Comment:
I'm pretty sure this is just because of how github renders those diffs - it literally is ` +\t` and thus the first tab is always "shorter". If it really were mixed tabs and spaces, the style checker would have complained.